docs/cloud/shared-infrastructure/faq.md

```diff
@@ -0,0 +1,79 @@
+---
+title: FAQ
+---
+
+## Preview
+
+During the preview, each organization can create up to two database instances at no cost.
+
+## Cluster provisioning
+
+#### Is it possible to change the cluster instance size or topology?
+
+Not yet, but it's on the roadmap for 2025.
+
+#### Are there plans to support manual resizing of cluster nodes?
+
+Yes, this is on the roadmap for 2025.
+
+#### Are there plans to support automatic resizing of cluster nodes?
+
+Yes, the goal is to support elastic scaling of the database cluster. This is on the roadmap for 2025.
+
+## Providers
+
+### Do you plan to support other Cloud providers?
+
+Absolutely! Both GCP and Azure are on the roadmap for 2025.
+
+## Managed KurrentDB
+
+#### Does the admin UI run on the provisioned cluster?
+
+Yes. Please refer to the [Cluster Addresses](ops/README.md#addresses-tab) section.
+
+## Backup and restore
+
+#### Is this possible?
+
+Not yet, but it's on the roadmap for 2025.
+
+## Support
+
+Support is not provided during the preview.
+
+## Security
+
+#### Does Kurrent have a security policy?
+
+Kurrent, Inc. has security policies in place.
+
+If you have specific questions please [contact us](https://www.kurrent.io/contact)
```

Comment:
Should we say something a bit more specific here? I think it might be best to explicitly state

> We've built Shared Infrastructure with security as an uncompromising requirement from the very first steps of designing the product. While Kurrent Cloud is ISO 27001 and SOC 2 certifications, the Shared Infrastructure Preview has not been included in those certifications. We will be including it in our next audit, but if you have a specific requirement for ISO 27001 or SOC 2, you should look to our Dedicated Infrastructure product.